[CN-894] Add note about the recommended port name (#776)
hazelcast/hz-docs: docs/modules/kubernetes/pages/kubernetes-auto-discovery.adoc

    port: 5701
```

> NOTE: If the Service is exposing multiple ports, it is recommended that you name the port that is exposing Hazelcast member as `hazelcast` in order for auto-discovery to function correctly.

Reviewer: ```suggestion
NOTE: If the service exposes multiple ports, name the port that exposes the Hazelcast member `hazelcast`. This configuration ensures that auto-discovery functions correctly.
```
Just minor rewording.
